cmd/ssd/ssd.py

```diff
@@ -14,6 +14,14 @@
     r'(25[0-5]|2[0-4][0-9]|[01]?[0-9]?[0-9])'
 )
 
+def which(name, defaultPath=""):
+    if defaultPath and os.path.exists(defaultPath):
```

Comment:
Instead of defining it, can we use this: https://docs.python.org/dev/library/shutil.html#shutil.which

Reply:
@fcrisciani right, I initially went this route, but `shutil.which` is only available in python 3.0 and has not been back-ported to 2.7, which is the version shipped in the ssd container and in the support dump container.

Reply:
sounds good